### ISSUE TITLE
TTT HUD tries to get the ammo clip of the spectator target

### ISSUE THREAD

**Issue by itsmeaddof123** [meepen/tttrw#88]
https://github.com/meepen/tttrw/blob/5cb7b4950b6b02b5a683db8c59eb9f3b2491f3e2/gamemode/vgui/ttt_hud_customizable.lua#L192

https://github.com/meepen/tttrw/blob/5cb7b4950b6b02b5a683db8c59eb9f3b2491f3e2/gamemode/vgui/ttt_hud_customizable.lua#L219

Whenever a dead player is spectating someone else, the TTT Ammo HUD tries to get their clip via Clip1 but that value isn't shared across clients. I believe the same issue happens with GetAmmoCount.

Also happens [here](https://github.com/meepen/tttrw/blob/5cb7b4950b6b02b5a683db8c59eb9f3b2491f3e2/gamemode/vgui/ttt_hud_customizable.lua#L254) and [here](https://github.com/meepen/tttrw/blob/5cb7b4950b6b02b5a683db8c59eb9f3b2491f3e2/gamemode/vgui/ttt_hud_customizable.lua#L269) and [here](https://github.com/meepen/tttrw/blob/5cb7b4950b6b02b5a683db8c59eb9f3b2491f3e2/gamemode/vgui/ttt_hud_customizable.lua#L273)

**Comment by meepen** [comment]
yeah the problem is that it's not networked to other clients, the only way to truly solve this is to make our own ammo network var on weapons.
